cmake/Helperfunc.cmake

```diff
@@ -20,37 +20,58 @@ function(add_exe name)
    set(exe_libraries "")
    set(exe_definitions "")
    set(exe_include_dirs "")
+
    foreach(arg IN LISTS ARGN)
-      if (arg STREQUAL "LIBRARIES")
+      if(arg STREQUAL "LIBRARIES")
```

Comment:
We should probably settle on one style or the other, right?  Without any of your changes (just looking at main):
```
sanchda@dev:~/dev/ddprof$ grep 'if (' CMakeLists.txt | wc -l
8
sanchda@dev:~/dev/ddprof$ grep 'if(' CMakeLists.txt | wc -l
16
```

Similarly,
```
sanchda@dev:~/dev/ddprof_metrics$ grep -nri 'if (' cmake/ | wc -l
7
sanchda@dev:~/dev/ddprof_metrics$ grep -nri 'if(' cmake/ | wc -l
80
```

It doesn't have to happen in this PR, but maybe we can agree that Cmake control flow parameter braces should "touch" the statement name?  `if()` not `if ()`?

Reply:
I agree !
I might just have done a `Format Document` in VS Code (which is provided by [CMake Language Support](https://marketplace.visualstudio.com/items?itemName=josetr.cmake-language-support-vscode) extension I think).
I also found another extension ([cmake-format](https://marketplace.visualstudio.com/items?itemName=cheshirekow.cmake-format)) based on a CLI tool, which might be better suited (since we could run cmake formatting checks in CI build).